emulator.c

```diff
@@ -155,8 +172,14 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
         break;
       }
     case 0x31: // NOLINT
-      printf("LXI SP");
-      break;
+      // printf("LXI SP");
```

Comment:
I like the comment indicating which instruction this is, but my personal preference is that you remove the printf part call -- makes it look like commented out code instead of an intentional comment!